### PR TITLE
Convert for-of into c-style for loops

### PR DESCRIPTION
**Pull request by akaplo**
Fixs showstopping bugs on any browser that doesn't support ES6 for loop constructs.

### CODE REVIEW

File: www/pages/route/route-controller.js [hunk 1]
```diff
@@ -92,12 +100,6 @@ angular.module('pvta.controllers').controller('RouteController', function($scope
     });
   };
 
-  var getHeart = function () {
```
Comment (akaplo) [inline www/pages/route/route-controller.js:95]: moved up

File: www/pages/route/route-controller.js [hunk 2]
```diff
@@ -32,26 +39,26 @@ angular.module('pvta.controllers').controller('RouteController', function($scope
...
+    $scope.route = route;
     getHeart();
     prepareStops($scope.route.Stops);
     $scope.vehicles = $scope.route.Vehicles;
     $ionicLoading.hide();
   });
 
   function prepareStops (stops) {
-    $scope.stops = []
+    $scope.stops = [];
```
Comment (akaplo) [inline www/pages/route/route-controller.js:50]: Indentation and semicolons are the only changes in this function

**Review by bgregg — APPROVED** [review]
👍

**Comment by akaplo** [comment]
CC issue is whatever.

**Pull request merged**